common4j/src/main/com/microsoft/identity/common/java/authorities/AzureActiveDirectoryAudience.java

```diff
@@ -154,6 +160,25 @@ public void setTenantId(String tenantId) {
         mTenantId = tenantId;
     }
 
+    public static String getAudienceFromAuthority( final AzureActiveDirectoryAuthority authority) {
+
+        if (authority == null || StringUtil.isNullOrEmpty(authority.getAudience().getTenantId())) {
+            return UNKNOWN_AUDIENCE;
+        }
+
+        final String audienceToCheck = authority.getAudience().mTenantId.toLowerCase(Locale.ROOT);
```

Comment:
What if `mTenantId` is null?

Reply:
for that we have a check isNullOrEmpty on the line above